### PR TITLE
modifying the pwl cost function to work with PSI

### PR DESCRIPTION
**Pull request by sourabhdalvi**

### CODE REVIEW

File: test/test_cost_functions.jl [hunk 1]
```diff
@@ -1,7 +1,7 @@
 using PowerSystems
 cost = VariableCost([(1.0, 1.0), (2.0, 1.1), (3.0, 1.2)])
 slopes = get_slopes(cost)
-res = [1.0, 10.0, 10.0]
+res = [10.0, 10.0]
```
Comment (jd-lara) [inline test/test_cost_functions.jl:4]: did you change it to return n - 1 slopes? Look at the docstring. I purposely made it for it to return n slopes. What's the reason for the change?

Comment (sourabhdalvi) [reply test/test_cost_functions.jl]: Sorry about, I did change it back to normal

**Comment by sourabhdalvi** [comment]
> please explain the update.
